website/docs/enterprise/getting-started/install-enterprise.mdx

```diff
@@ -137,33 +121,828 @@ Your private GitHub repo should have a clusters/management folder that includes
+bound to the `flux-system` namespace (where Flux stores its resources by default).
+The default set of rules are configured like this:
+
+```yaml
+rules:
+  # Flux Resources
+  - apiGroups: ["source.toolkit.fluxcd.io"]
+    resources: [ "buckets", "helmcharts", "gitrepositories", "helmrepositories", "ocirepositories" ]
+    verbs: [ "get", "list", "watch", "patch" ]
+
+  - apiGroups: ["kustomize.toolkit.fluxcd.io"]
+    resources: [ "kustomizations" ]
+    verbs: [ "get", "list", "watch", "patch" ]
+
+  - apiGroups: ["helm.toolkit.fluxcd.io"]
+    resources: [ "helmreleases" ]
+    verbs: [ "get", "list", "watch", "patch" ]
+
+  - apiGroups: [ "notification.toolkit.fluxcd.io" ]
+    resources: [ "providers", "alerts" ]
+    verbs: [ "get", "list", "watch", "patch" ]
+
+  - apiGroups: ["infra.contrib.fluxcd.io"]
+    resources: ["terraforms"]
+    verbs: [ "get", "list", "watch", "patch" ]
+
+  # Read access for all other Kubernetes objects
+  - apiGroups: ["*"]
+    resources: ["*"]
+    verbs: [ "get", "list", "watch" ]
+```
+
+These permissions give the emergency user Administrator-level powers. **We do not
+advise leaving it active on production systems**.
+
+If required, the permissions can be expanded with the `rbac.additionalRules` field in the
+[Helm Chart](../../references/helm-reference.md).
+Follow the instructions in the next section in order to configure RBAC correctly.
+
+:::tip
+To remove the emergency user as a login method, set the following values in the
+[Helm Chart](../../references/helm-reference.md):
+
+```yaml
+#
+adminUser:
+  create: false
+#
+additionalArgs:
+- --auth-methods=oidc
+#
+```
+
+If you are disabling an already existing emergency user, you will need to
+manually delete the Kubernetes Secret and any User Roles that were created on
+the cluster.
+:::
+
+</TabItem>
+</Tabs>
+
+### GitOps Dashboard Service Account Permissions
+
+This section covers the service account [permissions](https://kubernetes.io/docs/reference/access-authn-authz/rbac/)
+for the Weave GitOps application, which the WGE UI requires to work. The default permissions are defined in the [Helm chart](https://github.com/weaveworks/weave-gitops/tree/main/charts/gitops-server/templates/role.yaml), which will generate a cluster role with the following permissions:
+
+```yaml
+rules:
+# Used to query the cluster
+- apiGroups: [""]
+  resources:  ["users", "groups"]  # set by rbac.impersonationResources
+  verbs: [ "impersonate" ]
+  # resourceNames: []              # set by rbac.impersonationResourceNames
+# Used to get OIDC/static user credentials for login
+- apiGroups: [""]
+  resources: [ "secrets" ]
+  verbs: [ "get", "list" ]
+  resourceNames:                  # set by rbac.viewSecretsResourceNames
+    - "cluster-user-auth"
+    - "oidc-auth"
+# The service account needs to read namespaces to know where it can query
+- apiGroups: [ "" ]
+  resources: [ "namespaces" ]
+  verbs: [ "get", "list" ]
+```
```

Comment:
We don't let you configure this very much inn EE yet =/, (the comments here don't apply), and the EE SA has a few more permissions for `GitopsCluster` and `Cluster`. It looks like this if we wanna include it here.

```yaml
rules:
- apiGroups: [""]
  resources: ["users", "groups"]
  verbs: ["impersonate"]
- apiGroups:
  - ""
  resources:
  - secrets
  verbs:
  - get
  - list
  - watch
- apiGroups:
  - cluster.x-k8s.io
  resources:
  - clusters
  verbs:
  - get
  - list
  - watch
- apiGroups:
  - gitops.weave.works
  resources:
  - gitopsclusters
  verbs:
  - create
  - delete
  - get
  - list
  - patch
  - update
  - watch
- apiGroups:
  - gitops.weave.works
  resources:
  - gitopsclusters/finalizers
  verbs:
  - update
- apiGroups:
  - gitops.weave.works
  resources:
  - gitopsclusters/status
  verbs:
  - get
  - patch
  - update
```

Reply:
@foot Made the changes based on the above -- maybe take a quick look to make sure it's typo-free?